### DISCUSSION TITLE
Enhance Your GrapesJS Projects with ChatGPT Integration

### DISCUSSION THREAD

**Discussion by sathudeva7** [GrapesJS/grapesjs#5893] [Show and tell]
Hi, I am excited to share my contribution to the GrapesJS community. With this integration, users can now generate text content using AI directly within GrapesJS. I have successfully integrated ChatGPT with GrapesJS, making it easier to create dynamic and engaging content effortlessly.

Link : [https://www.npmjs.com/package/grapesjs-gpt-plugin](https://www.npmjs.com/package/grapesjs-gpt-plugin)
Demo: [https://jsfiddle.net/sathudeva7/ybjh4cwr/4/](https://jsfiddle.net/sathudeva7/ybjh4cwr/4/)
Vide Demo: [https://www.loom.com/share/061d84e2724d4ed597a9e8e1edf28aee?sid=ef8dccee-269a-490d-9f17-f3cf9e3e7b52](https://www.loom.com/share/061d84e2724d4ed597a9e8e1edf28aee?sid=ef8dccee-269a-490d-9f17-f3cf9e3e7b52)

**Comment by illtellyoulater** [comment]
@sathudeva7 great addition! 
Is it presently in a working state? 
Why not adding it to grapesjs community plugins page @ https://github.com/topics/grapesjs-plugin ?

**Comment by lexoyo** [comment]
Hello
Is it just a way to generate text right?
What about using function calling to let it use grapesjs API?

**Reply by sathudeva7** [reply]
yes we can check this, i changed the text color : https://www.loom.com/share/bdb3cd709672418f9dc3fd36c1e5a2a0?sid=795343f3-0f62-43e2-8c72-6768f6b3098c ,  and we can also create html blocks https://www.loom.com/share/2ca7e9116f4745dabdec7a7e1a9792ce?sid=9a6baf65-fd78-4965-ae55-0e9cef08aa14
